modules/video/test/test_trackers.cpp

```diff
@@ -64,40 +64,67 @@ TEST_P(DistanceAndOverlap, GOTURN)
 
 INSTANTIATE_TEST_CASE_P(Tracking, DistanceAndOverlap, TESTSET_NAMES);
 
-TEST(GOTURN, memory_usage)
+static bool checkIOU(const Rect& r0, const Rect& r1, double threshold)
 {
-    cv::Rect roi(145, 70, 85, 85);
+    int interArea = (r0 & r1).area();
+    double iouVal = (interArea * 1.0 )/ (r0.area() + r1.area() - interArea);;
 
+    if (iouVal > threshold)
+        return true;
+    else
+    {
+        std::cout << cv::format("Unmatched IOU:  expect IOU val (%lf) > the IOU threadhold (%lf)! Box 0 is ",
+                                iouVal, threshold)<< r0 <<", and Box 1 is "<<r1<< std::endl;
```

Comment:
It makes sense to avoid mess of C-style format strings and std::cout C++ approach.

Reply:
fixed.